package.json

```diff
@@ -19,6 +19,7 @@
     "@mui/icons-material": "^6.1.6",
     "@mui/material": "^6.1.6",
     "@tanstack/react-query": "^5.59.19",
+    "axios": "^1.7.7",
```

Comment:
I recommend to also install @types/axios as a dev package.

Reply:
<img width="1197" alt="Screenshot" src="https://github.com/user-attachments/assets/a1660e70-84b4-432a-be2f-df1ecf716506">

Reply:
did you get any eslint errors in axios.config file?

Reply:
getting "unsafe member access on an `any` type". but the type or `error` is `any` even inside axios type definitions:
<img width="921" alt="axios" src="https://github.com/user-attachments/assets/02b00d20-f56e-415d-8792-ff464cab17f0">
we will come back to this interceptor later anyway as there aren't any protected routes on the backend to test against..